diff --git b/apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1.Snippets/AssetServiceClientSnippets.g.cs a/apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1.Snippets/AssetServiceClientSnippets.g.cs
index 131f76293..2cc45eb39 100644
--- b/apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1.Snippets/AssetServiceClientSnippets.g.cs
+++ a/apis/Google.Cloud.Asset.V1Beta1/Google.Cloud.Asset.V1Beta1.Snippets/AssetServiceClientSnippets.g.cs
@@ -18,6 +18,7 @@ namespace Google.Cloud.Asset.V1Beta1.Snippets
 {
     using Google.Api.Gax;
     using Google.Api.Gax.Grpc;
+    using Google.Api.Gax.ResourceNames;
     using apis = Google.Cloud.Asset.V1Beta1;
     using Google.LongRunning;
     using Google.Protobuf;
