diff --git b/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2.Snippets/IncidentServiceClientSnippets.g.cs a/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2.Snippets/IncidentServiceClientSnippets.g.cs
index 29e5f1a8d..0892d6303 100644
--- b/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2.Snippets/IncidentServiceClientSnippets.g.cs
+++ a/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2.Snippets/IncidentServiceClientSnippets.g.cs
@@ -23,7 +23,7 @@ namespace Google.Cloud.Irm.V1Alpha2.Snippets
     using Google.Protobuf;
     using Google.Protobuf.WellKnownTypes;
     using Grpc.Core;
-    using SearchSimilarIncidentsResponse.Types;
+    using static SearchSimilarIncidentsResponse.Types;
     using System;
     using System.Collections;
     using System.Collections.Generic;
diff --git b/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2/IncidentServiceClient.cs a/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2/IncidentServiceClient.cs
index 54635ebce..cb450d01b 100644
--- b/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2/IncidentServiceClient.cs
+++ a/apis/Google.Cloud.Irm.V1Alpha2/Google.Cloud.Irm.V1Alpha2/IncidentServiceClient.cs
@@ -20,7 +20,6 @@ using gaxres = Google.Api.Gax.ResourceNames;
 using pb = Google.Protobuf;
 using pbwkt = Google.Protobuf.WellKnownTypes;
 using grpccore = Grpc.Core;
-using st = SearchSimilarIncidentsResponse.Types;
 using sys = System;
 using sc = System.Collections;
 using scg = System.Collections.Generic;
diff --git b/apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1.Snippets/SecurityCenterClientSnippets.g.cs a/apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1.Snippets/SecurityCenterClientSnippets.g.cs
index e52ec80f4..3e82acb81 100644
--- b/apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1.Snippets/SecurityCenterClientSnippets.g.cs
+++ a/apis/Google.Cloud.SecurityCenter.V1/Google.Cloud.SecurityCenter.V1.Snippets/SecurityCenterClientSnippets.g.cs
@@ -24,8 +24,8 @@ namespace Google.Cloud.SecurityCenter.V1.Snippets
     using Google.Protobuf;
     using Google.Protobuf.WellKnownTypes;
     using Grpc.Core;
-    using ListAssetsResponse.Types;
-    using ListFindingsResponse.Types;
+    using static ListAssetsResponse.Types;
+    using static ListFindingsResponse.Types;
     using System;
     using System.Collections;
     using System.Collections.Generic;
