diff --git b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Snippets/ClusterManagerClientSnippets.g.cs a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Snippets/ClusterManagerClientSnippets.g.cs
index d14d66c9c..8f0b68670 100644
--- b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Snippets/ClusterManagerClientSnippets.g.cs
+++ a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Snippets/ClusterManagerClientSnippets.g.cs
@@ -14,6 +14,11 @@
 
 // Generated code. DO NOT EDIT!
 
+// Ignore obsolete members within this file.
+// This is currently a hand-written addition to the generated file, until the
+// generator includes it automatically.
+#pragma warning disable CS0612 // Type or member is obsolete
+
 namespace Google.Cloud.Container.V1.Snippets
 {
     using Google.Api.Gax;
diff --git b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Tests/ClusterManagerClientTest.g.cs a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Tests/ClusterManagerClientTest.g.cs
index 35813ce7d..0acdfb11c 100644
--- b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Tests/ClusterManagerClientTest.g.cs
+++ a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1.Tests/ClusterManagerClientTest.g.cs
@@ -14,6 +14,11 @@
 
 // Generated code. DO NOT EDIT!
 
+// Ignore obsolete members within this file.
+// This is currently a hand-written addition to the generated file, until the
+// generator includes it automatically.
+#pragma warning disable CS0612 // Type or member is obsolete
+
 using gaxgrpc = Google.Api.Gax.Grpc;
 using wkt = Google.Protobuf.WellKnownTypes;
 using grpccore = Grpc.Core;
diff --git b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1/ClusterManagerClient.cs a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1/ClusterManagerClient.cs
index a780039a4..5819599b7 100644
--- b/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1/ClusterManagerClient.g.cs
+++ a/apis/Google.Cloud.Container.V1/Google.Cloud.Container.V1/ClusterManagerClient.g.cs
@@ -14,6 +14,11 @@
 
 // Generated code. DO NOT EDIT!
 
+// Ignore obsolete members within this file.
+// This is currently a hand-written addition to the generated file, until the
+// generator includes it automatically.
+#pragma warning disable CS0612 // Type or member is obsolete
+
 using gax = Google.Api.Gax;
 using gaxgrpc = Google.Api.Gax.Grpc;
 using proto = Google.Protobuf;
